### ISSUE TITLE
A11y_30days_Welcome to 30days of learning_Non Text content: “Alt” text is provided for “</>” decorative image.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#174]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:

#A11yTCS;#A11yMAS;#A11ySev3;#30days;#P3_30days_Dec22;#WCAG1.1.1;#DesktopWeb;#Win11;ChromiumEdge

### Environment Details:
Application Name: 30days
#URL: https://microsoft.github.io/30daysof/ 
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)
### Repro Steps:
1.	Hit the URL: https://microsoft.github.io/30daysof/
2.	“30Days of Learning” page will open.
3.	Tab to “</>#30Days of link” in header section and press F12.
4.	Observe that “Alt” text is provided for decorative images.
### Actual:
“Alt” text “30Days of Logo” is provided for “</>” decorative image.
### Expected:
“Alt” text should not be provided for “</>” decorative image, alt attribute should be null for decorative images.
### User Impact:
Screen reader users will face difficulty if “alt” text is provided for decorative images.
### Attachment:
<img width="960" alt="3_A11y_  WCP-CELAP3 30days_Welcome to 30days of learning_Devtools Alt text is provided for decorative image" src="https://user-images.githubusercontent.com/85863895/209327807-7e56f156-b5de-42c8-b667-bcbdb61d30fa.png">

**Comment by nitya** [comment]
@Ashutoshsahoo164 can you verify if issue is resolved?

**Comment by Raghu-addepalli** [comment]
![sw](https://user-images.githubusercontent.com/91586743/220274848-de6cc88f-b6c6-4d06-8e14-e27f4f236968.png)
Verified the issue in below environment issue is fixed hence closing the bug and added attachment.
Githubtags:  #Closed;
